src/utils/simpleRpcProvider.ts

```diff
@@ -2,13 +2,16 @@ import { ethers } from "ethers";
 
 const RPC_URL = "https://nodes.pancakeswap.info";
 const ETH_RPC_URL = "https://cloudflare-eth.com";
+const POLYGON_ZKEVM_RPC_URL = "https://f2562de09abc5efbd21eefa083ff5326.zkevm-rpc.com/";
```

Comment:
should we remove the `/` after `.com` ?

Reply:
<img width="762" alt="CleanShot 2023-07-26 at 13 47 37@2x" src="https://github.com/pancakeswap/token-list/assets/99634186/74aa60e4-0753-4ac3-a747-448e25ca6520">
I copy it from FE codebase, I can try it later